web/src/components/CleanLink.tsx

```diff
@@ -1,13 +1,21 @@
-import React, { ReactNode } from 'react'
+import React, { ReactNode, ReactElement } from 'react'
 import { Link } from 'react-router-dom'
 import styled from 'styled-components'
 
+import dimensions from '../constants/dimensions'
 import { isExternalUrl } from '../utils/UrlCheck'
 
 const StyledCleanLink = styled(Link)`
   color: inherit;
   text-decoration: none;
   display: flex;
+  align-items: center;
+  padding-top: 4px;
+  font-size: clamp(0.55rem, 1.6vh, ${props => props.theme.fonts.hintFontSize});
+
+  @media ${dimensions.smallViewport} {
+    gap: 8px;
+  }
```

Comment:
This should not be applied here because it would change the styles for all usages. Instead you can do ```
``` ts
const Link = styled(CleanLink)`
... styles
```

Reply:
True - sorry about that. Moved the styles back to their own component. The issue I previously mentioned about the theme prop not being accessible was also resolved by this.